cocoa/src/toga_cocoa/widgets/button.py

```diff
@@ -32,7 +32,7 @@ def create(self):
         self._icon = None
 
         self.native.buttonType = NSMomentaryPushInButton
-        self._set_button_style()
```

Comment:
This was getting called when Button's implementation is created, but it presupposes the existence of the button's style — which isn't there until Button calls `super.__init__()`. It also gets called when the button's font, bounds, or icon are set, and commenting it out here doesn't affect any testbed tests, so it seems it's still getting sufficiently applied. (I'm betting there are a few spots like this in the other implementation layers, though.)

Reply:
With the reordering I just did, this line no longer causes an issue. However... since leaving it out doesn't have any observable effect and the tests all pass, just leave it out I guess?